pyproject.toml

```diff
@@ -1,3 +1,34 @@
+[project]
+name = "readme_renderer"
+version = "41.0"
+description = '''readme_renderer is a library for rendering "readme" ' "descriptions for Warehouse'''
+authors = [
+  {email = "admin@mail.pypi.org"},
+  {name = "The Python Packaging Authority"}
+]
+readme = "README.rst"
+license = {file = "LICENSE"}
+dependencies = ["bleach>=2.1.0", "docutils>=0.13.1", "Pygments>=2.5.1"]
+dynamic = ["classifiers"]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Can we move the classifiers inline to pyproject.toml and remove the `dynamic` nature key here as well as the `[tool.setuptools.dynamic]` section? This reduces an extra file in the root of the project, as well as one setuptools-specific behavior.

See https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#classifiers for more